src/main/java/seedu/address/logic/commands/FavouriteCommand.java

```diff
@@ -0,0 +1,83 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Favourites an existing client in the address book by their index number
+ */
+public class FavouriteCommand extends Command {
+    public static final String COMMAND_WORD = "favourite";
+    public static final String MESSAGE_FAVOURITE_PERSON_SUCCESS =
+            "Favourited Client %1$s! Check if he/she exists in the Favourite Window!";
+    public static final String MESSAGE_UNFAVOURITE_PERSON_SUCCESS =
+            "Unfavourited Client %1$s! Check that he/she is removed from the Favourite Window!";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Favourites a client selected\n"
```

Comment:
Minor cosmetic detail but I noticed that if the String `COMMAND_WORD:  Favourites a client selected` is very long, then appending a `\n` to break up sentences would be necessary. In this case, a newline break mid sentence `"Favorites a client selected by the index number used in the last client listing."` seems odd. 

For example, `MESSAGE_USAGE` in `AddCommand` is quite short `add: Adds a person to the address book. Parameters: n/NAME p/PHONE e/EMAIL a/ADDRESS t/USERTYPE...` hence the inclusion of `\n` only after `Parameters: …`

Reply:
Thanks for the suggestion. I have made necessary changes.